pkgs/servers/pulseaudio/default.nix

```
@@ -108,7 +108,10 @@ stdenv.mkDerivation rec {
     rm -rf $out/{bin,share,etc,lib/{pulse-*,systemd}}
     sed 's|-lltdl|-L${libtool.lib}/lib -lltdl|' -i $out/lib/pulseaudio/libpulsecore-${version}.la
   ''
-    + ''moveToOutput lib/cmake "$dev" '';
+    + ''
+    moveToOutput lib/cmake "$dev"
+    rm -f $out/bin/qpaeq # this is packaged by the "qpaeq" package now, because of missing deps
```

Is not non-force  removal enough?

```suggestion
    rm $out/bin/qpaeq # this is packaged by the "qpaeq" package now, because of missing deps
```

---

The problem is that if libSupport is enabled it removes all of bin first, so it will fail without -f

---

If that's ok resolve conversation, otherwise comment